graphql-dxm-provider/src/main/java/org/jahia/modules/graphql/provider/dxm/node/GqlJcrNode.java

```diff
@@ -269,6 +269,11 @@ String getAggregatedLastModifiedDate(@GraphQLName("language") @GraphQLDescriptio
     List<String> getLanguagesToTranslate(@GraphQLName("languagesTranslated") @GraphQLDescription("List of known translated languages, will be used to compare modifications dates") List<String> languagesTranslated,
                                    @GraphQLName("languagesToCheck") @GraphQLDescription("List of languages potentially to be translated") List<String> languagesToCheck);
 
+    @GraphQLField
+    @GraphQLName("translationLanguages")
+    @GraphQLDescription("Returns nodes of available translations for this node")
```

Comment:
it should say return language of translated node

Reply:
We might also want to pass a boolean to be able to filter list against active languages on the site

Reply:
Added isActiveOnly param to filter active languages only; also added tests for both scenarios (all an active only)